app.json

```diff
@@ -3,7 +3,6 @@
     "name": "WayFinder",
     "slug": "expo-init-project",
     "privacy": "public",
-    "sdkVersion": "36.0.0",
```

Comment:
Should this be removed?

Reply:
Expo removed this line when I did `expo update`. I'm not sure what the point of it is, but I guess it makes expo use the latest sdk

Reply:
Ah, I see